homeassistant/components/hive/binary_sensor.py

```diff
@@ -114,6 +114,9 @@ async def async_update(self) -> None:
         self.device = await self.hive.sensor.getSensor(self.device)
         self.attributes = self.device.get("attributes", {})
         self._attr_is_on = self.device["status"]["state"]
+        if "status" in self.device:
```

Comment:
If a device is offline, it should be marked as unavailable instead.

Reply:
Thanks I have updated my commit to use the ```self._attr_available```